website/pages/docs/agent/autoauth/methods/kerberos.mdx

```diff
@@ -0,0 +1,31 @@
+---
+layout: "docs"
+page_title: "Vault Agent Auto-Auth Kerberos Method"
+sidebar_title: "Kerberos"
+sidebar_current: "docs-agent-autoauth-methods-kerberos"
+description: |-
+  Kerberos Method for Vault Agent Auto-Auth
+---
+
+# Vault Agent Auto-Auth Kerberos Method
+
+The `kerberos` agent provides an automated mechanism to retrieve
+a Vault token for Kerberos entities. It reads in configuration and
+identification information from the surrounding environment, and uses
+it to authenticate to Vault.
+
+For more on this auth method, see the [Kerberos auth method](https://www.vaultproject.io/docs/auth/kerberos.html).
```

Comment:
Should this be a relative path?  When we do doc previews it will go to the website otherwise.

Reply:
Oh, right. Yes, I will figure out how to do that..... :-)

Reply:
Fixed in https://github.com/hashicorp/vault/pull/8220/commits/0366cb65b374a556b36b1669810976c76a61b41f